bedrock/mozorg/templates/mozorg/home/home.html

```diff
@@ -84,20 +84,30 @@
 <header id="masthead">
   <div class="container">
     {% block tabzilla_tab %}
-    <div id="tabzilla">
-      <a href="{{ url('mozorg.home') }}">mozilla</a>
-    </div>
+      <div id="tabzilla">
+        <a href="{{ url('mozorg.home') }}">Mozilla</a>
+      </div>
+    {% endblock %}
+
+    {% block site_header_nav %}
```

Comment:
Is the home page navigation now identical to the one in `base-resp.html`, or does it differ? If it's now the same, is this still needed?